<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/pic/title.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>登陆管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setTimeButton">
       <property name="text">
        <string>设置考勤时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>CommList</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serialPortList"/>
     </item>
     <item>
      <widget class="QPushButton" name="refreshSerialPortList">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openCloseSerialPort">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="ClockInWidget" name="clockInPage">
      <attribute name="title">
       <string>Clock in</string>
      </attribute>
     </widget>
     <widget class="ManageWidget" name="manPage">
      <attribute name="title">
       <string>Manage</string>
      </attribute>
     </widget>
     <widget class="SelectWidget" name="scanPage">
      <attribute name="title">
       <string>scan</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClockInWidget</class>
   <extends>QWidget</extends>
   <header>ClockInWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ManageWidget</class>
   <extends>QWidget</extends>
   <header>ManageWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectWidget</class>
   <extends>QWidget</extends>
   <header location="global">selectwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
